diff --git a/node_modules/i18next/typescript/t.d.ts b/node_modules/i18next/typescript/t.d.ts
index 75aa72a..f5fd077 100644
--- a/node_modules/i18next/typescript/t.d.ts
+++ b/node_modules/i18next/typescript/t.d.ts
@@ -196,7 +196,7 @@ type _StripFormatOptions<F> = F extends `${infer Name}(${string})` ? Name : F;
 type _ResolveEntryType<Name extends string, Format> = [Format] extends [never]
   ? Name extends 'count'
     ? number
-    : string
+    : string | number
   : Format extends keyof _InterpolationFormatTypeMap
     ? _InterpolationFormatTypeMap[Format]
     : _StripFormatOptions<Format> extends keyof _InterpolationFormatTypeMap
