diff --git a/node_modules/partysocket/dist/chunk-2ESZ2MDZ.js b/node_modules/partysocket/dist/chunk-2ESZ2MDZ.js
index 374a95a..6c50f86 100644
--- a/node_modules/partysocket/dist/chunk-2ESZ2MDZ.js
+++ b/node_modules/partysocket/dist/chunk-2ESZ2MDZ.js
@@ -451,7 +451,7 @@ const partysocket = new PartySocket({
       }
       this._removeListeners();
       try {
-        if (this._ws.readyState === this.OPEN) {
+        if (this._ws.readyState === this.OPEN || this._ws.readyState === this.CONNECTING) {
           this._ws.close(code, reason);
         }
         this._handleClose(new Events.CloseEvent(code, reason, this));
diff --git a/node_modules/partysocket/dist/chunk-S74YV6PU.mjs b/node_modules/partysocket/dist/chunk-S74YV6PU.mjs
index 1837440..769909c 100644
--- a/node_modules/partysocket/dist/chunk-S74YV6PU.mjs
+++ b/node_modules/partysocket/dist/chunk-S74YV6PU.mjs
@@ -413,7 +413,7 @@ const partysocket = new PartySocket({
     }
     this._removeListeners();
     try {
-      if (this._ws.readyState === this.OPEN) {
+      if (this._ws.readyState === this.OPEN || this._ws.readyState === this.CONNECTING) {
         this._ws.close(code, reason);
       }
       this._handleClose(new Events.CloseEvent(code, reason, this));
